client/src/lib/dependency-injection/FormValidationManager.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * An API for updating form validation
+ */
+class FormValidationManager {
+  /**
+   * Constructor
+   * @param {object} values
+   */
+  constructor(values) {
+    this.values = values;
+    this.errorMap = {};
+  }
+
+  /**
+   * Adds an error to a given field
+   * @param {string} fieldName
+   * @param {string} message
+   * @returns {FormValidationManager}
+   */
+  addError(fieldName, message) {
+    if (!this.fieldExists(fieldName)) {
+      throw new Error(`Tried to add error to non-existent field: ${fieldName}`);
```

Comment:
Getting this error a lot in behat.

Reply:
https://travis-ci.org/silverstripe/silverstripe-campaign-admin/jobs/317896254

Reply:
Have pushed a PR for this: https://github.com/silverstripe/silverstripe-admin/pull/387